src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt

```diff
@@ -3,28 +3,56 @@ package org.jetbrains.research.transformations.util
 import org.apache.commons.io.FileUtils
 import org.jetbrains.research.transformations.util.ParserSetup.checkSetup
 import java.io.File
+import java.io.InputStream
+import java.net.URL
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.StandardCopyOption
 import java.util.logging.Logger
 
+
 /**
  * [ParserSetup] class is created for parser setup before using Gumtree with python code.
  * Also [checkSetup] should be called before running tests.
  */
 object ParserSetup {
-    private val LOG = Logger.getLogger(javaClass.name)
+    private val logger = Logger.getLogger(javaClass.name)
+
+    private const val PARSER_REPOSITORY_ZIP_URL = "https://github.com/elena-lyulina/pythonparser/archive/master.zip"
```

Comment:
I think we should change this link since the repo was moved to the Jetbrains Research